tslint.json

```diff
@@ -0,0 +1,116 @@
+{
+  "rules": {
+    "align": [
+      true,
+      "statements"
+    ],
+    "ban": [
+      false
+    ],
+    "class-name": true,
+    "comment-format": [
+      true,
+      "check-space"
+    ],
+    "curly": true,
+    "eofline": true,
+    "indent": [
+      true,
+      "spaces"
+    ],
+    "interface-name": [
+      true,
+      "always-prefix"
+    ],
+    "jsdoc-format": true,
+    "max-line-length": [
+      false,
+      140
+    ],
+    "semicolon": [
+      true,
+      "always"
+    ],
+    "member-access": false,
+    "member-ordering": [
+      false,
+      { "order": "fields-first" }
+    ],
+    "new-parens": true,
+    "no-arg": false,
+    "no-default-export": true,
+    "no-duplicate-key": true,
+    "no-duplicate-variable": true,
+    "no-inferrable-types": false,
+    "no-shadowed-variable": false,
+    "no-trailing-whitespace": true,
+    "no-unreachable": false,
+    "no-unused-expression": false,
+    "no-unused-variable": true,
+    "no-use-before-declare": true,
+    "one-line": [
+      true,
+      "check-open-brace",
+      "check-catch",
+      "check-else",
+      "check-finally",
+      "check-whitespace"
+    ],
+    "one-variable-per-declaration": [
+      false,
+      "ignore-for-loop"
+    ],
+    "quotemark": [
+      true,
+      "single"
+    ],
+    "trailing-comma": [
+      false,
+      {
+        "multiline": "always",
+        "singleline": "never"
+      }
+    ],
+    "typedef": [
+      true,
+      "property-declaration"
+    ],
+    "typedef-whitespace": [
+      true,
+      {
+        "call-signature": "nospace",
+        "index-signature": "nospace",
+        "parameter": "nospace",
+        "property-declaration": "nospace",
+        "variable-declaration": "nospace"
+      },
+      {
+        "call-signature": "space",
+        "index-signature": "space",
+        "parameter": "space",
+        "property-declaration": "space",
+        "variable-declaration": "space"
+      }
+    ],
+    "use-isnan": true,
+    "use-strict": [
+      false,
+      "check-module",
+      "check-function"
+    ],
+    "variable-name": [
+      false,
+      "check-format",
+      "allow-leading-underscore",
+      "ban-keywords"
+    ],
+    "whitespace": [
+      true,
+      "check-branch",
+      "check-decl",
+      "check-operator",
+      "check-separator",
+      "check-type"
+    ]
+  }
+}
```

Comment:
Yes, but it requires contributors to know that and setup their IDE V.S. the default tslint file. Yes it's duplicated, but I think it's a good think to simplify devs life.